runtime/moonbase/src/governance/tracks.rs

```diff
@@ -35,13 +35,13 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 8]
 			// Amount that must be placed on deposit before a decision can be made.
```

Comment:
In the cummunity forum, capacity for this track is 5

Reply:
I've just discussed with Yann and we will copy the Moonriver setup to Moonbase, I'll update soon